iceoryx_hoofs/test/moduletests/test_cxx_algorithm.cpp

```diff
@@ -37,7 +38,7 @@ class algorithm_test : public Test
         std::string output = testing::internal::GetCapturedStderr();
         if (Test::HasFailure())
         {
-            std::cout << output << std::endl;
+            IOX_LOG(ERROR) << output;
         }
```

Comment:
Please remove output capturing. It it not required anymore with the new logger since all the output is captured by default and only on error the messages are logged to the console. See also #998